tools/bump_ocp_releases.py

```diff
@@ -62,6 +62,7 @@
 OCP_REPLACE_CONTEXT = ['"{version}"', "ocp-release:{version}"]
 
 SKIPPED_MAJOR_RELEASE = ["4.6"]
+MAJOR_MINOR_VERSION_REGEX = re.compile("^([1-9]\d*|0)(\.(([1-9]\d*)|0))$")
```

Comment:
lint probably wants
```suggestion
MAJOR_MINOR_VERSION_REGEX = re.compile(r"^([1-9]\d*|0)(\.(([1-9]\d*)|0))$")
```

Reply:
btw, doesn't it work too?
```suggestion
MAJOR_MINOR_VERSION_REGEX = re.compile("^[1-9][0-9]*\.[1-9][0-9]*$")
```

Reply:
> lint probably wants

probably, changed.

Reply:
> btw, doesn't it work too?

it won't match 5.0 for example

Reply:
right